src/Recruit.Vacancies.Client/Infrastructure/Client/VacancyClient.cs

```diff
@@ -98,8 +102,25 @@ public Task UpdateVacancyAsync(Vacancy vacancy)
             };
 
             await _messaging.SendCommandAsync(command);
+            await UpdateDashboardAsync(vacancy.EmployerAccountId);
 
             return true;
         }
+        
+        public async Task<Dashboard> GetDashboardAsync(string employerAccountId)
+        {
+            var key = string.Format(QueryViewKeys.DashboardViewPrefix, employerAccountId);
```

Comment:
Do we need a prefix?

Reply:
Again, I am not sure I did this because otherwise we may have other views for an employer that isn't data to do with a dashboard and this makes it seem a bit more key/valuey even though it isn;t plain key/value.

Reply:
the idea of the composite key (type + subject) is that it avoids us having to create separate collections for every different type of projection as there will be lots of them in the app further down the line. it's also inherently unique so could be used as the document id as chris suggested elsewhere in this pr.

i'd also originally thought we should add a "type" field in there so that we could (for example) retrieve all the vacancy projects. i suspect we will end up needing this but isn't strictly needed right now.